cg/services/order_validation_service/workflows/balsamic/models/sample.py

```diff
@@ -17,15 +14,15 @@ class BalsamicSample(Sample):
     control: ControlEnum | None = None
     elution_buffer: ElutionBuffer | None = None
     formalin_fixation_time: int | None = None
-    is_tumour: bool
     phenotype_groups: list[str] | None = None
     phenotype_terms: list[str] | None = None
     post_formalin_fixation_time: int | None = None
-    reference_genome: GenomeVersion
+    reference_genome: GenomeVersion | None = None
```

Comment:
This should just be removed right?

Reply:
I think so. Currently it is not recommended to run Balsamic with other than hg19. Also, I don't see any option on the order portal where customers could specify the reference genome